css/index.css

```diff
@@ -98,3 +98,36 @@ header a:not(:last-child) {
     align-items: center;
 }
 
+.img-section-2 {
+    background-image: url(/Users/maksim/Desktop/project/images/photo%20secondary%20section.png);
+    background-repeat: no-repeat;
+    background-position: right top;
+}
+
+.container-section-2 {
+    max-width: 1110px;
+    width: 100%;
+    margin: 0 auto;
+    height: 742px;
+}
+
+.section-2-content {

```

Comment:
section-2 - bad name. after some time we can move this block, and class we be wrong named

Reply:
and still bad name - 'after some time we can move this block, and class we be wrong named'